tests/phpunit/tests/media.php

```diff
@@ -3617,6 +3619,69 @@ private function reset_omit_loading_attr_filter() {
 		// Clean up the above filter.
 		remove_filter( 'wp_omit_loading_attr_threshold', '__return_null', 100 );
 	}
+
+	/**
+	 * Test the wp_default_image_output_mapping function.
+	 *
+	 * @ticket 55443
+	 */
+	public function test_wp_default_image_output_mapping() {
+		$mapping = wp_default_image_output_mapping( array() );
+		$this->assertSame( array( 'image/jpeg' => 'image/webp' ), $mapping );
+	}
+
+	/**
+	 * Test that wp_default_image_output_mapping doesn't overwrite existing mappings.
+	 *
+	 * @ticket 55443
+	 */
+	public function test_wp_default_image_output_mapping_existing() {
+		$mapping = array( 'mime/png' => 'mime/webp' );
+		$mapping = wp_default_image_output_mapping( $mapping );
+		$this->assertSame(
+			array(
+				'mime/png'   => 'mime/webp',
+				'image/jpeg' => 'image/webp',
+			),
+			$mapping
+		);
+	}
+
+	/**
+	 * Test that the image editor default output for JPEGs is WebP.
+	 *
+	 * @ticket 55443
+	 */
+	public function test_wp_image_editor_default_output_maps_to_webp() {
+		remove_filter( 'image_editor_output_format', '__return_empty_array' );
+
+		$editor = wp_get_image_editor( DIR_TESTDATA . '/images/canola.jpg' );
+		if ( is_wp_error( $editor ) ) {
+			$this->markTestSkipped( $editor->get_error_message() );
+		}
+		$resized = $editor->resize( 100, 100, false );
+
+		if ( is_wp_error( $resized ) ) {
+			$this->markTestSkipped( $resized->get_error_message() );
+		}
+
+		$dest_file = $editor->generate_filename();
+		$saved     = $editor->save( $dest_file );
+
+		if ( is_wp_error( $saved ) ) {
+			$this->markTestSkipped( $saved->get_error_message() );
+		}
```

Comment:
Shouldn't the `markTestSkipped` calls here rather become failures? If any of these functions/methods return an error, we should probably be alerted about that. So I would suggest to use `assertNotWPError` for all of them.
```suggestion
		$this->assertNotWPError( $editor );

		$resized = $editor->resize( 100, 100, false );
		$this->assertNotWPError( $resized );

		$dest_file = $editor->generate_filename();
		$saved     = $editor->save( $dest_file );
		$this->assertNotWPError( $saved );
```

Reply:
Right, that should be better. Applying.